run

```diff
@@ -177,8 +204,8 @@ case $run_command in
 
     # Run watch command in the background
     if [[ ${1:-} == "--watch" ]] || [[ ${1:-} == "-w" ]]; then
-      container_name="yarn-$(date +'%s')"
-      trap 'docker kill ${container_name}' EXIT
```

Comment:
I've tested this without this line, and it does, as I suspected, leave the watcher running:

``` bash
$ docker ps -q
$ ./run start --watch
** run the server**
** ctrl + c **
$ docker ps -q
d56ab6af95e8
```

We need this line.